server/twitch.handler.ts

```diff
@@ -1,15 +1,14 @@
 import * as tmi from 'tmi.js';
+import { Subscription } from 'rxjs';
+import { PersistenceInstance } from './persistence';
+import { startWith } from 'rxjs/operators';
+import { Twitch, TwitchEventTypes, TwitchTriggerCommand } from '../projects/contracts/src/lib/types';
+import { triggerMediaClipById } from './websocket-server';
 import {EmoteObj} from 'tmi.js';
-import {Subscription} from "rxjs";
-import {PersistenceInstance} from "./persistence";
-import {startWith} from "rxjs/operators";
-import {Twitch, TwitchTriggerCommand} from "../projects/contracts/src/lib/types";
-import {triggerMediaClipById} from "./websocket-server";
 import {Logger} from 'winston';
 import {newLogger} from "./logger.utils";
 
-
-declare module "tmi.js" {
+declare module 'tmi.js' {
```

Comment:
could you extract this to a different `d.ts` file?

Reply:
Where should I store that file ? Within the `server` folder or create a `/server/types` folder first and then create `index.d.ts` file in there?

Reply:
server/types should be fine, just for this type of "declare" it should be in a `d.ts` file then it will be "found automagically"

Reply:
yeah it doesnt work like that sadly with these dts